test/unit/android-helper-specs.js

```diff
@@ -80,6 +78,15 @@ describe('Android Helpers', () => {
     it('should fail if avd name is not specified', async () => {
       await helpers.prepareEmulator(adb, {}).should.eventually.be.rejected;
     });
+    it('should set the correct avdArgs', async () => {
```

Comment:
This should be in a separate `describe` block for `prepareAVDArgs`, not tacked onto the one for `prepareEmulator`.